service/storage_proxy.cc

```diff
@@ -2906,10 +2906,13 @@ storage_proxy::storage_proxy(distributed<replica::database>& db, storage_proxy::
     , _cancellable_write_handlers_list(std::make_unique<cancellable_write_handlers_list>()) {
     namespace sm = seastar::metrics;
     _metrics.add_group(storage_proxy_stats::COORDINATOR_STATS_CATEGORY, {
-        sm::make_queue_length("current_throttled_writes", [this] { return _throttled_writes.size(); },
-                       sm::description("number of currently throttled write requests")),
+        sm::make_current_bytes("view_update_backlog", [this] { return _max_view_update_backlog.fetch_shard(this_shard_id()).current; },
+                       sm::description("View update backlog cached size")),
+    });
+    _metrics.add_group(storage_proxy_stats::REPLICA_STATS_CATEGORY, {
+        sm::make_current_bytes("view_update_backlog", [this] { return _max_view_update_backlog.fetch_shard(this_shard_id()).current; },
+                       sm::description("View update backlog cached size")),
```

Comment:
Why did you remove the current_throttled_writes statistics? Did you remove it by mistake?
And what's the difference between the two metrics you added here? Maybe part of the same mistake?

Also, please expand on the description "View update backlog cached size". I have no idea what this means, and neither will users.

Reply:
I replaced the `current_throttled_writes ` by an extra repetition of the new metric by mistake, sorry.
I added this metric mostly to enable testing this change, but I improved the description slightly in the rebase anyway